test/k8sT/manifests/demo_ds.yaml

```diff
@@ -13,6 +13,17 @@ spec:
       labels:
         zgroup: testDS
     spec:
+      # Some tests (e.g. DSR) requires to send requests to testds from a host
+      # which does not run any of its pods. We've chosen k8s3 to be such host.
+      affinity:
```

Comment:
This file is used for many tests. I would strongly suggest to have a new file dedicated for the tests that can only be run with +3 VMs. Can be done in a follow up PR.

Reply:
Will do it as a follow up!

Reply:
Please create a GH issue so we can track this.

Reply:
Actually just did in this PR.